### ISSUE TITLE
Showing fake AP's but not deauthing

### ISSUE THREAD

**Issue by ghost** [sidward35/Deauth-RickRollAP#1]
Just compiled using latest android IDE with patched libraries for esp8266 , so far it's showing the rick roll ap's but not deauthing.

I selected the proper wifi channel in the code. The other deauther script this code is based off of works fine for me... now sure why this isn't working

Am I missing something?  Does this script deauth 1 time only? then transmit rick roll beacons there after?

**Comment by sidward35** [comment]
Nope, you're not missing anything! This code is still buggy, and isn't completely functional as of now. I haven't been able to get the ESP8266 to send deauth frames and set up fake APs simultaneously (it's either one or the other, which is why the deauther script this is based off works just fine), and I've put this project on hold for a while. But I am aware of this issue and it's the first thing on my to-do for this project, so I'll try to get to it as soon as I can.
